src/routes/controllers/users.controller.js

```diff
@@ -93,6 +93,7 @@ export async function getAuthorProfile(req, res) {
 export const updateUserProfile = async (req, res) => {
   const { id } = req.user;
   const { firstName, lastName, username, bio, image } = req.body;
+  const user = await getUserbyId(id);
```

Comment:
Why was this line added?

Reply:
I used it to test the documentation. forgot to remove it.